Coding/server/src/WiFiManager/.gitignore

```diff
@@ -0,0 +1 @@
+APCredentials.h
```

Comment:
This issue has been fixed. A seperate header file was used to not disclose the used passphrase but because it is supposed to be public anyways, it has been moved into the WiFiManager class.